oak_functions_containers_container/build_container_bundle

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# A simple script to create a tarball of an example OCI Runtime Bundle (https://github.com/opencontainers/runtime-spec/blob/4e3b9264a330d094b0386c3703c5f379119711e8/bundle.md)
+# that can be executed on Oak Containers. Used for manual testing during
+# development. Down the line we probably want to replace this.
+
+set -o errexit
+
+readonly OCI_IMAGE_FILE="./target/oak_container_example_oci_image.tar"
+set -e
+rm --recursive --force ./target
+mkdir --parents ./target
+
+cargo build \
+    --package="oak_functions_containers_app" \
```

Comment:
I don't think these quotes do anything here, may as well remove them

Reply:
I have removed those quotes, but I do think the quotes in general help with readability of the script.

Reply:
SG, though FYI quotes have a specific purpose in bash scripts (interpolation, whitespace escaping, etc.), they are not usually used just to make things more readable

Reply:
I'm going to change my tune on this comment. Going to remove the quotes. This makes the formatting more or less consistent with the ```README.md``` bash commands.

For scripts going forward I will:
1) Only rely on quotes when needed.
2) Use equal signs for all inputted arguments